src/net_processing.cpp

```diff
@@ -1734,17 +1602,6 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
         pfrom->fSuccessfullyConnected = true;
     }
 
-    else if (!pfrom->fSuccessfullyConnected 
-            // HACK to allow 0.13.* peers to send us MNVERIFY without sending VERACK first
```

Comment:
only the additional `if` condition (`&& strCommand != NetMsgType::MNVERIFY`) is up for removal and not the entire statement. It should be
```
    if (!pfrom->fSuccessfullyConnected) {
        // Must have a verack message before anything else
        LOCK(cs_main);
        Misbehaving(pfrom->GetId(), 1);
        return false;
    }

```

Reply:
Yes, thanks!